pkg/core/interop_system.go

```diff
@@ -139,6 +139,26 @@ func bcGetTransaction(ic *interop.Context, v *vm.VM) error {
 	return nil
 }
 
+// bcGetTransactionFromBlock returns transaction with the given index from the
+// block with height or hash specified.
+func bcGetTransactionFromBlock(ic *interop.Context, v *vm.VM) error {
+	hash, err := getBlockHashFromElement(ic.Chain, v.Estack().Pop())
+	if err != nil {
+		return err
+	}
+	block, err := ic.Chain.GetBlock(hash)
+	if err != nil || !isTraceableBlock(ic, block.Index) {
+		v.Estack().PushVal(stackitem.Null{})
+	}
+	index := v.Estack().Pop().BigInt().Int64()
+	if index < 0 || index >= int64(len(block.Transactions)) {
```

Comment:
C# implementation uses `len-1` (and `index+1` below). Probably 1st tx shouldn't be taken into account.

Reply:
That's true.